GDJS/Runtime/events-tools/commontools.ts

```diff
@@ -458,6 +458,36 @@ namespace gdjs {
       export const trunc = function (x: float): number {
         return x | 0;
       };
+
+      /**
+       * Compute the X position when given an angle and distance relative to the starting position.
+       * This is also known as getting the cartesian coordinates of a 2D vector, using its polar coordinates.
+       * @param positionX The position on X-axis, in pixels.
+       * @param angle The angle, in degrees.
+       * @param distance The distance from the object, in pixels.
+       */
+      export const XFromAngleAndDistance = function (
+        positionX: float,
```

Comment:
Let's remove the positionX argument :) Usually these expressions just take angle/distance as the origin is considered by default.

Reply:
This is the free expression, we need a starting position.
Or i miss something?

Reply:
@4ian This is the free expression, the expression based object is on the other file. See my other comment.

Reply:
@Silver-Streak What do you think? Usually in these mathematical/vector helper functions, 0;0 is always considered as the origin. I'm just a bit afraid of `XFromAngleAndDistance(100, 45, 100)` being more cryptic than `100 + XFromAngleAndDistance(45, 100)`.

Reply:
Ooohh okey I see what you mean now, i agree with you `100 + XFromAngleAndDistance(45, 100)` is better, I'll remove `position`.